src/components/dialogs/restore-modification-dialog.js

```diff
@@ -106,6 +131,23 @@ const RestoreModificationDialog = ({
                                 ref={provided.innerRef}
                                 {...provided.droppableProps}
                             >
+                                <Box sx={styles.selectAll}>
+                                    <Checkbox
+                                        color={'primary'}
+                                        edge="start"
+                                        checked={
+                                            selectedItems.size ===
+                                            modificationsToRestore.length
```

Comment:
I believe a renaming is in order, given that we can now select stashed modifications to either restore or permanently delete them. My suggestion is to replace it by "stashedModifications"